packages/stark-ui/src/modules/toast-notification/toast-notification.module.ts

```diff
@@ -0,0 +1,42 @@
+import { NgModule, ModuleWithProviders, SkipSelf, Optional } from "@angular/core";
+import { BrowserModule } from "@angular/platform-browser";
+import { MatButtonModule, MatIconModule, MatSnackBarModule } from "@angular/material";
+import { TranslateModule } from "@ngx-translate/core";
+import { STARK_TOAST_NOTIFICATION_SERVICE, StarkToastNotificationServiceImpl } from "./services";
+import { StarkToastNotificationComponent } from "./components";
+
+@NgModule({
+	declarations: [StarkToastNotificationComponent],
+	imports: [BrowserModule, MatButtonModule, MatIconModule, MatSnackBarModule, TranslateModule],
+	exports: [StarkToastNotificationComponent],
+	entryComponents: [StarkToastNotificationComponent]
```

Comment:
Why using entryComponents here ?

Reply:
Because component isn't used as is (no reference in the template). 
We use the service to open the toast, which then uses the component.
See this : https://github.com/angular/material2/issues/3002